clog/loggers.py

```diff
@@ -303,7 +303,7 @@ def __init__(self):
         self.lines = {}
 
     def log_line(self, stream, line):
-        assert isinstance(line, six.text_type)
+        assert isinstance(line, (bytes, six.string_types))
```

Comment:
Okay, I assume stream name can only be string?

Tried to switch tests/test_clog.py::TestFileLogger::test_log_line[hello-hello\nworld\n] to log a string casted to bytes, and it broke in py34... So I'm assuming that this means it's not a legal type.